core/domain/opportunity_services.py

```diff
@@ -86,12 +87,30 @@ def get_exploration_opportunity_summary_from_model(model):
     new_incomplete_translation_language_codes = (
         model.incomplete_translation_language_codes + missing_language_codes)
 
+    translation_in_review_counts = {}
+    for language_code in constants.SUPPORTED_CONTENT_LANGUAGES:
```

Comment:
This probably needs to be done as a get-multi rather than lots of individual gets in a for loop. Otherwise it does not scale well with the number of langauges.

Reply:
Done